internal/tasks/manifests_test.go

```diff
@@ -547,24 +547,24 @@ func answerMostWith404(h http.Handler) http.HandlerFunc {
 
 func TestCheckVulnerabilitiesForNextManifest(t *testing.T) {
 	test.WithRoundTripper(func(_ *test.RoundTripper) {
-		j, s := setup(t, test.WithClairDouble)
+		j, s := setup(t, test.WithClairDouble, test.WithTrivyDouble)
 		s.Clock.StepBy(1 * time.Hour)
 
 		//setup two image manifests with just one content layer (we don't really care about
 		//the content since our Clair double doesn't care either)
 		images := make([]test.Image, 3)
 		for idx := range images {
 			images[idx] = test.GenerateImage(test.GenerateExampleLayer(int64(idx)))
-			images[idx].MustUpload(t, s, fooRepoRef, "")
+			images[idx].MustUpload(t, s, fooRepoRef, fmt.Sprint(idx))
```

Comment:
While not wrong, I found this choice of function surprising. A more conventional choice is `Itoa`.
```suggestion
			images[idx].MustUpload(t, s, fooRepoRef, strconv.Itoa(idx))
```